src/client/utils/url.js

```diff
@@ -1,297 +1,115 @@
-import sharedUrlUtil from '../../utils/url';
-import { isCrossDomainWindows } from '../utils/dom';
-import { getAttribute as nativeGetAttribute } from '../sandbox/native-methods';
-import { DOM_SANDBOX_STORED_ATTR_POSTFIX, DOCUMENT_CHARSET } from '../../const';
+import sharedUrlUtils from '../../utils/url';
+import originLocation from './origin-location';
+import { DOCUMENT_CHARSET } from '../../const';
 import { get as getSettings } from '../settings';
 
-var urlUtil = {};
+class UrlUtils {
```

Comment:
This shouldn't be a class. Utilities actually are never (for exception for EventEmitter) are classe. They don't have inner state, just utility methods. Just `export` methods directly.